README.rst

```diff
@@ -154,7 +154,7 @@ Developers". Feel free to cite our work in your research using the following Bib
 
 .. code-block::
 
-    @software{pygmt_2021_4592991,
+    @software{pygmt_2021_4978645,
       author       = {Uieda, Leonardo and
```

Comment:
> > > How do I know whether any of the current order reflects a case-by-case basis or whether I should update based on `git shortlog -sne`?
> > 
> > See previous discussion at [#1047 (comment)](https://github.com/GenericMappingTools/pygmt/pull/1047#discussion_r593880470). Will leave it up to you to take a first stab at the order :) Just to clarify, that line refers to the Zenodo citation order. The release note authors order should go strictly by `git shortlog` (and remove the bots).
> 
> OK, I guess I wasn't paying good attention for that release :) Thanks for the note.

Just on the author list, this is the order from `git shortlog -sne` on the master branch (truncated to be for authors with >2 commits):

```
   279  Dongdong Tian <seisman.info@gmail.com>
   256  Leonardo Uieda <leouieda@gmail.com>
   141  Wei Ji <23487320+weiji14@users.noreply.github.com>
    83  Will Schlitzer <schlitzer90@gmail.com>
    51  Michael Grund <23025878+michaelgrund@users.noreply.github.com>
    36  Meghan Jones <meghanj@hawaii.edu>
    31  Yao Jiayuan <coreman.seism@gmail.com>
    27  dependabot[bot] <49699333+dependabot[bot]@users.noreply.github.com>
    15  Liam Toney <38269494+liamtoney@users.noreply.github.com>
     3  kmaterna <kathrynmaterna@gmail.com>
     2  Abhishek Anant <81291713+itsabhianant@users.noreply.github.com>
     2  Andrey Shmakov <akshmakov@gmail.com>
     2  Conor Bacon <conor.bacon@gmail.com>
     2  MichaeINeumann <81390813+MichaeINeumann@users.noreply.github.com>
     2  Noor Buchi <55197145+noorbuchi@users.noreply.github.com>
     2  Shivani chauhan <56837749+xdshivani@users.noreply.github.com>
     2  Tyler Newton <54011699+tjnewton@users.noreply.github.com>
     2  alperen-kilic <58446832+alperen-kilic@users.noreply.github.com>
```

Might need to shuffle some positions. And we'll need to add in Abhishek since he's on the AUTHORS.md (added in #1157).

Reply:
Will update shortly.

Reply:
Shuffled in https://github.com/GenericMappingTools/pygmt/pull/1341/commits/c2c210df1eed9f84249ffb10b56efc4dde670f72.